sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetScanBase.scala

```diff
@@ -871,6 +872,28 @@ trait ParquetPartitionReaderBase extends Logging with Arm with ScanWithMetrics
     currentChunk
   }
 
+  /**
+   * Take case-sensitive into consideration when getting the data reading column names
+   * before sending parquet-format buffer to cudf.
```

Comment:
```suggestion
   * before sending Parquet-formatted buffer to cudf.
```

Also applies to other instance of this below.

Reply:
Done. Thx

Reply:
This comment was not addressed.  Is there a commit that has not been pushed to Github yet?

Reply:
It's strange, I just checked the source file and it should be changed to parquet-formatted

Reply:
That's why I asked if there's a commit that hasn't been pushed to Github yet.  According to https://github.com/NVIDIA/spark-rapids/pull/3982/files#diff-8fa3aa74be556181661b80e4735b00e8d453311293c30ccc6d92353df4636af9R877 it's not updated yet, and I also pulled the source locally to verify it's not some weird browser cached state problem with Github's rendering:
```
$ git fetch https://github.com/wbo4958/spark-rapids.git hive-issue
From https://github.com/wbo4958/spark-rapids
 * branch                hive-issue -> FETCH_HEAD
$ git co FETCH_HEAD
HEAD is now at 16c081d8c resolve comments
$ grep parquet-formatted sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetScanBase.scala 
   * before sending parquet-formatted buffer to cudf.
   * @param fileSchema the schema of the dumped parquet-formatted buffer
```